Emulator/SPD.File.Emulator/SpdEmulatorApi.cs

```diff
@@ -0,0 +1,135 @@
+﻿using SPD.File.Emulator.Interfaces;
+using SPD.File.Emulator.Interfaces.Structures.IO;
+using SPD.File.Emulator.Utilities;
+using FileEmulationFramework.Interfaces;
+using FileEmulationFramework.Interfaces.Reference;
+using FileEmulationFramework.Lib.Memory;
+using FileEmulationFramework.Lib.Utilities;
+using Microsoft.Win32.SafeHandles;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace SPD.File.Emulator;
+
+public class SpdEmulatorApi : ISpdEmulator
+{
+    private readonly IEmulationFramework _framework;
+    private readonly SpdEmulator _spdEmulator;
+    private readonly Logger _logger;
+
+    public SpdEmulatorApi(IEmulationFramework framework, SpdEmulator spdEmulator, Logger logger)
+    {
+        _framework = framework;
+        _spdEmulator = spdEmulator;
+        _logger = logger;
+    }
+    public bool TryCreateFromFileSlice(string sourcePath, long offset, string route, string destinationPath)
+    {
+        _logger.Info("[SpdEmulatorApi] TryCreateFromFileSlice: {0}, Ofs {1}, Route {2}", sourcePath, offset, route);
```

Comment:
There might be a bug here, seems the handle is not closed, it probably should be.

Affects all emulators with `TryCreateFromFileSlice` API. In this case, also affects `RegisterSpd` and `TryCreateFromSpd`.
Since they were copied from my AWB Emulator, it seems the culprit is... me :sweat_smile: 

This will need fixing in a separate PR for other projects, do fix it here though.

CC. @AnimatedSwine37